packages/core-components/src/components/table/table.tsx

```diff
@@ -23,6 +23,9 @@ export class TableComponent {
    **/
   @Prop() size: TableSizes = TableSizes.EXPAND;
 
+  /** Whether or not the column the header is associated with can be resized by dragging. Per default it is false. */
```

Comment:
Comment should state that this makes every column of the table resizable at once.